docs/AboutUs.md

```diff
@@ -2,58 +2,59 @@
 layout: page
```

Comment:
Can we remove these 2 lines? Seems like its just for boilerplate

![image](https://user-images.githubusercontent.com/78403168/221089825-aebbe396-b8da-4a76-a631-20b94b9d33a5.png)

Reply:
This is actually the configuration values for Jekyll, so Jekyll knows how to display the page in HTML